### PR TITLE
Helm charts for scylla-operator, scylla-manager and scylla

### LINKED ISSUE #42: Package up a helm chart
We should be able to create a general helm chart from the examples we have. This chart needs parametrization for resources and perhaps a switch for enabling monitoring.

### PR DESCRIPTION
**Pull request by zimnx**
Definition of two helm charts:
- scylla-operator - allows to install customizable Scylla Operator
- scylla-manager - allows to install customizable Scylla Manager 
- scylla - allows to create and customize Scylla clusters

Helm charts can be released to publicly available GCS bucket.

To add repository to your local Helm users can execute:

```
helm repo add scylla-operator https://storage.googleapis.com/scylla-operator-charts
```

And then all available charts will be installable from CLI:

```
$ helm search repo scylla
NAME                            CHART VERSION   APP VERSION     DESCRIPTION                                       
scylla-operator/scylla          1.0.0           v1.0.0          Scylla is a close-to-the-hardware rewrite of Ca...
scylla-operator/scylla-manager  1.0.0           v1.0.0          Scylla Manager automates database operations.     
scylla-operator/scylla-operator 1.0.0           v1.0.0          Scylla Operator is a Kubernetes Operator for ma...

```

**Which issue is resolved by this Pull Request:**
Fixes #42 

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [x] Image has been built (`make docker-build`) on the last commit.

### CODE REVIEW

File: helm/scylla-operator/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+The Scylla Operator has been installed. Check its status by running:
+
+  kubectl -n {{ .Values.namespace }} get pods -l "app.kubernetes.io/name={{ template "scylla-operator.name" . }}"
+
+Visit https://github.com/scylladb/scylla-operator for tutorials on how to
+create and configure Scylla clusters using the Scylla Operator and set up monitoring.
```
Comment (markallanson) [inline helm/scylla-operator/templates/NOTES.txt:7]: Would be nice here to have the notes also include a port forward command for hooking up locally

**Review by markallanson — COMMENTED** [review]
I don't have any experience running scylla yet, but this looks like just what we are after for our evaluation - Thanks!

File: helm/scylla-operator/templates/NOTES.txt (hunk 1 above)
Comment (zimnx) [reply helm/scylla-operator/templates/NOTES.txt:7]: Currenly we support only cases where application using Scylla is deployed on k8s next to Scylla. This doesn't mean it's not possible, although we don't have enough generic solution to include it here. So I thought adding a link to repo itself where we also have docs, would be the best here.

Comment (markallanson) [reply helm/scylla-operator/templates/NOTES.txt:7]: I was thinking more for debugging and development. Doing a simple port forward should work for this

```
kubectl port-forward --namespace <the-namespace> svc/scylla 9042:9042
```

Putting this in the notes would be helpful as a copy/paste

**Comment by markallanson** [comment]
Are there plans to add support for basic `nodeSelector` for scylla nodes?

**Comment by zimnx** [comment]
> Are there plans to add support for basic `nodeSelector` for scylla nodes?

Currently you can use taints and tolerations or set up a node afiinity.

**Pull request merged**